### PR TITLE
[1623] Validate numericality of course fees

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Revert of https://github.com/DFE-Digital/manage-courses-backend/pull/338/files

### Changes proposed in this pull request
Ensure we validate numerically of uk and international fees

### Guidance to review
Try and submit a non-numerical character

### Checklist
- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Add missing validations to enrichments](https://trello.com/c/e7wGJYfZ/1623-add-missing-validations-to-enrichments)

**Comment by fofr** [comment]
We need to allow empty fees when saving:
https://trello.com/c/wAnoDLPH/1672-incorrect-fee-validation-when-saving-different-type-of-enrichment

We need to allow empty international fees when publishing.

File: spec/requests/api/v2/providers/courses/update_spec.rb [hunk 1]
```diff
@@ -266,8 +266,8 @@ def perform_request(course)
         {
           about_course: "",
           fee_details: "",
-          fee_international: "",
-          fee_uk_eu: "",
+          fee_international: 0,
```
Comment (fofr) [inline spec/requests/api/v2/providers/courses/update_spec.rb]: This looks wrong to me. 0 has a different meaning to empty string.

Comment (dankmitchell) [reply spec/requests/api/v2/providers/courses/update_spec.rb]: Correct, but this is the same implementation as the c# app.

**Pull request merged**